mslib/msui/qt5/ui_performance_settings.py

```diff
@@ -1,130 +0,0 @@
-        font = QtGui.QFont()
-        font.setBold(False)
-        font.setWeight(50)
-        self.label_5.setFont(font)
-        self.label_5.setObjectName("label_5")
-        self.horizontalLayout.addWidget(self.label_5)
-        self.lbAircraftName = QtWidgets.QLabel(PerformanceSettingsDialog)
-        font = QtGui.QFont()
-        font.setBold(False)
-        font.setWeight(50)
-        self.lbAircraftName.setFont(font)
-        self.lbAircraftName.setObjectName("lbAircraftName")
-        self.horizontalLayout.addWidget(self.lbAircraftName)
-        self.pbLoadPerformance = QtWidgets.QPushButton(PerformanceSettingsDialog)
-        self.pbLoadPerformance.setObjectName("pbLoadPerformance")
-        self.horizontalLayout.addWidget(self.pbLoadPerformance)
-        self.verticalLayout_2.addLayout(self.horizontalLayout)
-        spacerItem1 = QtWidgets.QSpacerItem(0, 0, QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.MinimumExpanding)
-        self.verticalLayout_2.addItem(spacerItem1)
-        self.horizontalLayout_2 = QtWidgets.QHBoxLayout()
-        self.horizontalLayout_2.setObjectName("horizontalLayout_2")
-        self.label = QtWidgets.QLabel(PerformanceSettingsDialog)
-        font = QtGui.QFont()
-        font.setBold(False)
-        font.setWeight(50)
-        self.label.setFont(font)
-        self.label.setObjectName("label")
-        self.horizontalLayout_2.addWidget(self.label)
-        self.dteTakeoffTime = QtWidgets.QDateTimeEdit(PerformanceSettingsDialog)
-        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Minimum)
-        sizePolicy.setHorizontalStretch(0)
-        sizePolicy.setVerticalStretch(0)
-        sizePolicy.setHeightForWidth(self.dteTakeoffTime.sizePolicy().hasHeightForWidth())
-        self.dteTakeoffTime.setSizePolicy(sizePolicy)
-        self.dteTakeoffTime.setObjectName("dteTakeoffTime")
-        self.horizontalLayout_2.addWidget(self.dteTakeoffTime)
-        self.verticalLayout_2.addLayout(self.horizontalLayout_2)
-        self.horizontalLayout_4 = QtWidgets.QHBoxLayout()
-        self.horizontalLayout_4.setObjectName("horizontalLayout_4")
-        self.label_2 = QtWidgets.QLabel(PerformanceSettingsDialog)
-        font = QtGui.QFont()
-        font.setBold(False)
-        font.setWeight(50)
-        self.label_2.setFont(font)
-        self.label_2.setObjectName("label_2")
-        self.horizontalLayout_4.addWidget(self.label_2)
-        self.dsbTakeoffWeight = QtWidgets.QDoubleSpinBox(PerformanceSettingsDialog)
-        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Minimum)
-        sizePolicy.setHorizontalStretch(0)
-        sizePolicy.setVerticalStretch(0)
-        sizePolicy.setHeightForWidth(self.dsbTakeoffWeight.sizePolicy().hasHeightForWidth())
-        self.dsbTakeoffWeight.setSizePolicy(sizePolicy)
-        self.dsbTakeoffWeight.setDecimals(0)
-        self.dsbTakeoffWeight.setMaximum(9999999.0)
```

Comment:
Maximum/Decimals/Step is not set accordingly in new controls. They are upward limited to 99.99, which is wrong.

Reply:
I have fixed this in the recent commit. Also the fuel value had the same constrains, fixed that as well.